brainstormforce/ultimate-addons-for-gutenberg
# Migration logic nested blocks

### Description
This PR adds compatibility to Child Block in Frontend CSS generation.

### Screenshots
NA - The end output doesn't change.

### How has this been tested?
1. Create Icon List in master branch.
2. Checkout to this branch and you must see no CSS changes.
3. The migration should go smoothly.
